src/components/table/table.tsx

```diff
@@ -71,6 +71,12 @@ export class Table {
     @Event()
     public load: EventEmitter<TableParams>;
 
+    /**
+     * Emitted when a row is activated
+     */
+    @Event()
+    public activate: EventEmitter<any>;
```

Comment:
We should not use `any` here, it does not help the consumer in any way in knowing what data will be part of the event